Tests/PointFreeTests/__Snapshots__/NewslettersTests/testExpressUnsubscribeReply.1.email_settings_before_unsubscribe.txt

```diff
@@ -0,0 +1,9 @@
+▿ 2 elements
+  ▿ PointFree.Database.EmailSetting
+    - newsletter: PointFree.Database.EmailSetting.Newsletter.announcements
```

Comment:
this is a before snapshot of user's newsletter subscriptions before hitting the endpoint...